### PR TITLE
feat(router-logs): add command to enable/disable router-logs

### LINKED ISSUE #605: Add a command to activate router logs
The command could be named `router-logs`. Take example on the command `force-https`.

### PR DESCRIPTION
**Pull request by sihamais**
- [x] Add a changelog entry in the section "To Be Released" of CHANGELOG.md

Fix #605

### CODE REVIEW

**Comment by sihamais** [comment]
Why there isn't a description of `force-https` and `sticky-session` on the README ? Should I put one for router-logs ?

**Comment by EtienneM** [comment]
> Why there isn't a description of `force-https` and `sticky-session` on the README ? Should I put one for router-logs ?

I'm not sure I understand what you mean here. Can you elaborate a bit?

File: apps/routing_settings.go [hunk 1]
```diff
@@ -48,3 +49,25 @@ func StickySession(appName string, enable bool) error {
...
+	}
+
+	var action string
+	if enable {
+		action = "enable"
+	} else {
+		action = "disable"
+	}
+
+	io.Statusf("Router logs has been %sd on %s\n", action, appName)
```
Comment (EtienneM) [inline apps/routing_settings.go:71]: Isn't it how we are supposed to conjugate it? 🤔 
```suggestion
	io.Statusf("Router logs have been %sd on %s\n", action, appName)
```

**Review by EtienneM — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -101,6 +101,7 @@ COMMANDS:
      restart                 Restart processes of your app
      force-https
      sticky-session
```
Comment (sihamais) [inline README.md:103]: Here, these commands do not have a description on the README, I was wondering why ? @EtienneM

Comment (EtienneM) [reply README.md:103]: Oh I didn't noticed 😅 
I think it's missing. If you can add a description on these commands to that would be nice

File: apps/routing_settings.go (hunk 1 above)
Comment (sihamais) [reply apps/routing_settings.go:71]: Copied it from force-https :rofl:

File: README.md (hunk 2 above)
Comment (sihamais) [reply README.md:103]: I'm on it.

File: apps/routing_settings.go (hunk 1 above)
Comment (sihamais) [reply apps/routing_settings.go:71]: Found this : 
`If the subject of the sentence is He – She – It or a singular noun (car, bird, child) we use ‘has been‘.
If the subject of a sentence is I – You – We – They or a plural noun (cars, birds, children) we use ‘have been‘.`
So `has been` would be the answer.

Comment (EtienneM) [reply apps/routing_settings.go:71]: Isn't it the "logs" that are enabled? Hence "have been"?

File: README.md [hunk 3]
```diff
@@ -99,8 +99,9 @@ COMMANDS:
      ps                      Display your application containers
      scale, s                Scale your application instantly
      restart                 Restart processes of your app
-     force-https
-     sticky-session
+     force-https             Enable/Disable automatic redirection of traffic to HTTPS for your application
+     sticky-session          Enable/Disable sticky sessions for your application
```
Comment (EtienneM) [inline README.md:103]: These lines are just copy-paste of `scalingo help`. Hence you need to update the `Description` attributes of `force-https` and `sticky-session`. This is not just a README update.

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,7 @@
 
 ### To be Released
 
+* feat(router-logs): add command `router-logs` to enable/disable router-logs on your application [692](https://github.com/Scalingo/cli/pull/692)
```
Comment (EtienneM) [inline CHANGELOG.md:5]:
```suggestion
* feat(router-logs): add command `router-logs` to enable/disable router-logs on your application [#692](https://github.com/Scalingo/cli/pull/692)
```

**Review by EtienneM — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (sihamais) [reply README.md:103]: Do you want me to keep the README description or the command description ?
#### Example for force-https :
Command's description : 
`When enabled, this feature will automatically redirect HTTP traffic to HTTPS for all domains associated with this application.`

**VS**

README description :
`Enable/Disable automatic redirection of traffic to HTTPS for your application`

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
 ### To be Released
 
-* feat(router-logs): add command `router-logs` to enable/disable router-logs on your application [692](https://github.com/Scalingo/cli/pull/692)
+* feat(router-logs): add command `router-logs` to enable/disable router-logs on your application [#692](https://github.com/Scalingo/cli/pull/692)
```
Comment (EtienneM) [inline CHANGELOG.md:5]:
```suggestion
* feat(router-logs): add command `router-logs` to enable/disable router logs on your application [#692](https://github.com/Scalingo/cli/pull/692)
```

File: README.md (hunk 3 above)
Comment (EtienneM) [reply README.md:103]: Oh sorry, after double checking I think this is the `Usage` attribute that should be updated with the same text you currently have here (i.e. "Enable/Disable automatic redirection of traffic to HTTPS for your application")

**Review by EtienneM — COMMENTED** [review]
Sorry, just some small things

File: README.md (hunk 3 above)
Comment (sihamais) [reply README.md:103]: Oh okey I get it. Will do.

**Review by EtienneM — APPROVED** [review]

**Pull request merged**